modules/proxy/main.tf

```
@@ -4,61 +4,53 @@ locals {
 }
 
 resource "google_compute_health_check" "load_balancer_out" {
-  name = "${local.prefix}-lb-out-${var.install_id}"
+  name = "${local.prefix}-lb-out"
```

so for all the other prefixes, there's an assumption that the separator will be at the end of the `var.prefix`
but for this local prefix, it's not, for consistency do we want to move it to be 
```suggestion
  name = "${local.prefix}lb-out"
```

with 

```
locals {
  prefix = "${var.prefix}proxy-"
}
```

---

Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/58/commits/5cbe54c4e4636dcb984e110e6376c88b5b3cec2b.